go/test/endtoend/vtgate/foreignkey/fk_fuzz_test.go

```diff
@@ -123,22 +123,28 @@ func (fz *fuzzer) generateQuery() []string {
 	}
 }
 
+func getInsertType() string {
```

Comment:
nitpick - remove this all together?

Reply:
kept it so that easy to test on other branches as well.